<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget_STRAZH_IP</class>
 <widget class="QWidget" name="Widget_STRAZH_IP">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>550</width>
    <height>111</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>550</width>
     <height>111</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="My_lineEdit" name="Icon2Path" native="true">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>20</y>
      <width>280</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="My_lineEdit" name="Icon3Path" native="true">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>75</y>
      <width>280</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_87">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>55</y>
      <width>50</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Пароль</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_86">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>0</y>
      <width>47</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Логин</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_88">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>55</y>
      <width>231</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>IP-адрес поворотной платформы</string>
    </property>
   </widget>
   <widget class="My_lineEdit" name="Icon4Path" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>75</y>
      <width>261</width>
      <height>30</height>
     </rect>
    </property>
    <property name="inputMask" stdset="0">
     <string/>
    </property>
    <property name="clearButtonEnabled" stdset="0">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="my_combobox" name="Icon1Path">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>261</width>
      <height>30</height>
     </rect>
    </property>
    <property name="editable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_85">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>231</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>IP-адрес камеры</string>
    </property>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>my_combobox</class>
   <extends>QComboBox</extends>
   <header>my/my_combobox.h</header>
  </customwidget>
  <customwidget>
   <class>My_lineEdit</class>
   <extends>QWidget</extends>
   <header>my_lineedit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
